providers/implementations/ciphers/cipher_aes_cbc_hmac_sha.c

```diff
@@ -229,7 +229,8 @@ static int aes_get_ctx_params(void *vctx, OSSL_PARAM params[])
     }
     p = OSSL_PARAM_locate(params, OSSL_CIPHER_PARAM_IV);
     if (p != NULL
-        && !OSSL_PARAM_set_octet_string(p, ctx->base.oiv, ctx->base.ivlen)) {
+        && !OSSL_PARAM_set_octet_string(p, ctx->base.oiv, ctx->base.ivlen)
+        && !OSSL_PARAM_set_octet_ptr(p, &ctx->base.oiv, ctx->base.ivlen)) {
```

Comment:
Oooooh, I never thought of doing it this way!  Nice!

Reply:
I was just cribbing from https://github.com/openssl/openssl/blob/6985b0e3deaee2f6e83a670ce7b33cf9ee47933a/providers/implementations/ciphers/ciphercommon.c#L361-L367 , for which it seems we can blame ... you (in 06c8331c51797f43881e5262b4c114aacc6d0ca0) :)
Shane moved it around a bunch in follow-up commits, though.  (Also, passing `&ctx->iv` for both octet-string and octet-ptr feels weird to me.)

Reply:
Oh, uhm, I guess I've forgotten stuff I've done.